ouroboros-consensus-diffusion/test/consensus-test/Test/Consensus/PointSchedule.hs

```diff
@@ -464,21 +465,26 @@ uniformPoints BlockTree {btTrunk, btBranches} g = do
 newtype GenesisWindow = GenesisWindow { unGenesisWindow :: Word64 }
   deriving (Show)
 
+newtype ForecastRange = ForecastRange { unForecastRange :: Word64 }
+  deriving (Show)
+
 -- | All the data used by point schedule tests.
 data GenesisTest = GenesisTest {
   gtHonestAsc     :: Asc,
   gtSecurityParam :: SecurityParam,
   gtGenesisWindow :: GenesisWindow,
+  gtForecastRange :: ForecastRange, -- REVIEW: Do we want to allow infinite forecast ranges?
```

Comment:
I think it is fine to test with finite ones, as that is also the only thing we will use in production. We can still test with very large forecast ranges in any case.

Reply:
Yes, `ForecastRange maxBound` is virtually infinite.

Reply:
We might have to be a bit careful with `maxBound` in particular to avoid overflows though.

Reply:
True. Maybe `maxBound / 2` which is also virtually infinite?